Feature_Guide/Data_Pipelines/README.md

```diff
@@ -412,7 +414,7 @@ print(query_response.as_numpy("ensemble_output_FP32_image"))
 print(query_response.as_numpy("ensemble_output_bool"))
 ```
 
-The second part to look at is the general client API. This is the same as any Triton Client, simply use the python client API to query the server.
+The second part to look at is understanding the Triton client API.
```

Comment:
The next part is just running client/server. Where are we understanding the client API? Is there an actionable step for the user here? I still don't understand this line